pkg/plugins/resources/k8s/store_test.go

```diff
@@ -612,6 +635,40 @@ var _ = Describe("KubernetesStore", func() {
 				"k8s.kuma.io/namespace": ns,
 				"k8s.kuma.io/name":      "two",
 			}))
+
```

Comment:
If you've got longer tests `ginkgo` authors propose using `By()`
https://onsi.github.io/ginkgo/#documenting-complex-its-by

So it would be `By("listing resources from default mesh")`, `By("listing resources from demo mesh")`

Reply:
Got it, thanks!